src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 40 * days;
+  let discount = 0;
+
+  if (days > 6) {
+    discount = 50;
+  } else if (days > 2) {
```

Comment:
Change the condition to 'days >= 3' to include the case when days are exactly 3.